opentracing-api/src/main/java/io/opentracing/ActiveSpan.java

```diff
@@ -0,0 +1,71 @@
+package io.opentracing;
+
+import java.io.Closeable;
+
+/**
+ * In any execution context (or any thread, etc), there is at most one "active" {@link Span} primarily responsible for
+ * the work accomplished by the surrounding application code. That active Span may be accessed via the
+ * {@link ActiveSpanSource#active()} method. If the application needs to defer work that should be part of the same Span, the
+ * Source provides a {@link ActiveSpan#defer} method that returns a {@link Continuation}; this continuation may be used
+ * to re-activate and continue the {@link Span} in that other asynchronous executor and/or thread.
+ *
+ * <p>
+ * {@link ActiveSpan}s are created via {@link Tracer.SpanBuilder#startAndActivate()} or {@link ActiveSpanSource#adopt}. They can
+ * be {@link ActiveSpan#defer()}ed as {@link ActiveSpan.Continuation}s, then re-{@link Continuation#activate()}d later.
+ *
+ * @see ActiveSpanSource
+ */
+public interface ActiveSpan extends Closeable, Span {
+    /**
+     * Mark the end of the active period for the {@link Span} pinned by this {@link ActiveSpan}. When the last
+     * {@link ActiveSpan} is deactivated for a given {@link Span}, it is automatically {@link Span#finish()}ed.
+     * <p>
+     * <p>
+     * NOTE: It is an error to call deactivate() more than once on a single {@link ActiveSpan}.
+     *
+     * @see Closeable#close() {@link ActiveSpan}s are auto-closeable and may be used in try-with-resources blocks
+     */
+    void deactivate();
+
+    /**
+     * "Fork" a new {@link Continuation} associated with this {@link ActiveSpan} and {@link Span}, as well as any
+     * 3rd-party execution context of interest.
+     * <p>
+     * <p>
+     * The associated {@link Span} will not {@link Span#finish()} while a {@link Continuation} is outstanding; in
+     * this way, it provides a reference/pin just like an active @{Handle} does.
+     *
+     * @return a new {@link Continuation} to {@link Continuation#activate()} at the appropriate time.
+     */
+    Continuation defer();
```

Comment:
yes, I used to call this `capture()`... the thing I liked/like about `defer()` is that it clearly (to me) implies that the `Continuation` that's returned should actually get *used*. `capture()` felt (again, to me) like the caller could keep the ref or discard it as they pleased... the discard case is really bad since the Span leaks a reference and thus never terminates.